tests/device_selection/test_predictor_ml.py

```diff
@@ -49,19 +52,32 @@ def test_setup_device_predictor_with_prediction(path_uncompiled_circuits: Path,
             dump(qc, f)
 
     device = get_device("ibm_falcon_127")
-
-    success = setup_device_predictor(
-        devices=[device],
-        figure_of_merit="expected_fidelity",
-        path_uncompiled_circuits=path_uncompiled_circuits,
-        path_compiled_circuits=path_compiled_circuits,
-    )
+    if gnn:
+        success = setup_device_predictor(
+            devices=[device],
+            figure_of_merit="expected_fidelity",
+            path_uncompiled_circuits=path_uncompiled_circuits,
+            path_compiled_circuits=path_compiled_circuits,
+            gnn=True,
+            number_epochs=10,
+            number_trials=2,
+        )
+    else:
+        success = setup_device_predictor(
+            devices=[device],
+            figure_of_merit="expected_fidelity",
+            path_uncompiled_circuits=path_uncompiled_circuits,
+            path_compiled_circuits=path_compiled_circuits,
+        )
```

Comment:
You could just use these values as a default and further reduce complexity here.

```suggestion
    success = setup_device_predictor(
        devices=[device],
        figure_of_merit="expected_fidelity",
        path_uncompiled_circuits=path_uncompiled_circuits,
        path_compiled_circuits=path_compiled_circuits,
        gnn=gnn
    )
```